File: c++/Human.cpp
class Human {
    private:
        string __nik; // atribut NIK
        string __nama; // atribut Nama
        string __jenisKelamin; // Jenis Kelamin

    public:
        Human(string nik = "", string nama = "", string jenisKelamin = "") {
            // inisiasi dengan value awal
            __nik = nik;
            __nama = nama;
            __jenisKelamin = jenisKelamin;
        }

        string getNama() {
            // return nama human
            return __nama;
        }

        string getNik() {
            // return nik human
            return __nik;
        }

        string getJenisKelamin() {
            // return jenis kelamin
            return __jenisKelamin;
        }

        void setNama(string nama) {
            // mengganti value nama
            __nama = nama;
        }

        void setNik(string nik) {
            // mengganti value nik
            __nik = nik;
        }

        void setJenisKelamin(string jenisKelamin) {
            // mengganti value Jenis Kelamin
            __jenisKelamin = jenisKelamin;
        }
};

File: python/Human.py
class Human:
    __nik = '' # atribut NIK
    __nama = ''# atribut Nama
    __jenisKelamin = '' # Jenis Kelamin
    
    def __init__(self, nik= '', nama = '', jenisKelamin = ''):
      # inisiasi dengan value awal
      self.__nama = nama
      self.__nik = nik
      self.__jenisKelamin = jenisKelamin
    

    def getNama(self):
      # return nama human
      return self.__nama
    
    def getNik(self):
      # return nik human
      return self.__nik
    
    def getJenisKelamin(self):
      # rettun jenis kelamin
      return self.__jenisKelamin
    
    def setNama(self, nama):
      # mengganti value nama
      self.__nama = nama
    
    def setNik(self, nik):
      # mengganti value nik
      self.__nik = nik
    
    def setJenisKelamin(self, jenisKelamin):
      # mengganti value Jenis Kelamin
      self.__jenisKelamin = jenisKelamin
    
